src/Core/DB/Adapter.php

```diff
@@ -87,7 +87,7 @@ public function errno(): int;
     public function error(): ?string;
 
     /**
-     * Returns MySQL-styled error code (.....)
+     * Returns SQLSTATE error code
```

Comment:
Just "Driver-specific error code." as described here
![image](https://user-images.githubusercontent.com/35626465/113990559-05bb0f80-986b-11eb-9366-83b47ed4b4d8.png)

Reply:
`errorCode()` returns `PDOStatement::errorCode()` which results to element 0 in `PDOStatement::errorInfo()`, hence SQLSTATE (as described in docs).
`errno()` however returns element 1 which is driver-specific error code.

Maybe instead of returning SQLSTATE return driver-specific error code and deprecate `errno()`?